docs/.vitepress/vitepress/styles/content/table-of-content.scss

```diff
@@ -44,19 +44,22 @@
         margin-top: 10px;
         font-size: 11px;
         color: var(--text-color-lighter);
-        text-overflow: ellipsis;
-        overflow: hidden;
-        white-space: nowrap;
         color: inherit;
```

Comment:
```suggestion
        color: inherit;
        list-style: none;
```

Reply:
> 

Maybe it's my first time trying to commit to a review, but now it seems like I've changed it.